xml/tls_jarm.xml

```diff
@@ -120,9 +123,9 @@
     <description>VMWare ESXi</description>
     <example>21d14d00021d21d21c21d14d21d21d3e9a0dda94718e521eb7d1409c9e3601</example>
     <param pos="0" name="os.vendor" value="VMware"/>
-    <param pos="0" name="os.family" value="ESXi"/>
+    <param pos="0" name="os.family" value="VMware ESX/ESXi"/>
     <param pos="0" name="os.product" value="ESXi"/>
-    <param pos="0" name="hw.device" value="Hypervisor"/>
```

Comment:
We haven't used this value before but I do _like_ what it's saying about this endpoint. I'm on the fence on this one.  If we adopt it we will need to apply it across the board in our other fingerprints.  I can do this in this PR.
Any thoughts on the @gschneider-r7 @rkirk-r7 @hdm @dabdine-r7 ?